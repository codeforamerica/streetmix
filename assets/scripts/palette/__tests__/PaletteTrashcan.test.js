/* eslint-env jest */
import React from 'react'
import { shallow } from 'enzyme'
import { PaletteTrashcan } from '../PaletteTrashcan'

describe('PaletteTrashcan', () => {
  it('renders with the `-visible` class when `visible` prop is true', () => {
    const wrapper = shallow(<PaletteTrashcan visible />)
    expect(wrapper).toMatchSnapshot()
    expect(wrapper.find('.palette-trashcan-visible').length).toEqual(1)
  })

  it('doesnâ€™t render with the `show` class when `visible` prop is false', () => {
    const wrapper = shallow(<PaletteTrashcan visible={false} />)
    expect(wrapper.find('.palette-trashcan-visible').length).toEqual(0)
  })
})
