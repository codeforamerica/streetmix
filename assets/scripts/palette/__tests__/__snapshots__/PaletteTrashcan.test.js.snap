// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaletteTrashcan renders with the \`-visible\` class when \`visible\` prop is true 1`] = `
<div
  className="palette-trashcan palette-trashcan-visible"
>
  <FormattedMessage
    defaultMessage="Drag here to remove"
    id="palette.remove"
    values={Object {}}
  />
</div>
`;
